docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   proxy:
-    image: traefik:1.7.10-alpine
+    image: amd64/traefik:v2.5.0
```

Comment:
I have objections here. v2.5 is still an RC version. Im not objecting to using an RC version. We should just absolutely not be specifying a minor version. If the still expected bug to be comes along then the specification of the minor version ensure that we dont get the bug fix.

Reply:
Don't know where you got the info that it's an RC? It's a regular release here: https://github.com/traefik/traefik/releases/tag/v2.5.0

And I completely forgot about the minor version tag thing, you're right! I'll change that

Reply:
Yeah i stand corrected the release has been out for a full 2 days.

Reply:
Oh lol I didn't even look at the date. Fun! :p